src/clients/api/mutations/useRepayVai.ts

```diff
@@ -0,0 +1,29 @@
+import { MutationObserverOptions, useMutation } from 'react-query';
+
+import { repayVai, IRepayVaiInput, IRepayVaiOutput } from 'clients/api';
+import FunctionKey from 'constants/functionKey';
+import { useVaiUnitroller } from 'hooks/useContract';
+
+type Options = MutationObserverOptions<
+  IRepayVaiOutput,
+  Error,
+  Omit<IRepayVaiInput, 'vaiControllerContract'>
+>;
+
+const useRepayVai = (options?: Options) => {
+  const vaiControllerContract = useVaiUnitroller();
+
+  // @TODO: invalidate queries related to fetching the user VAI balance and
```

Comment:
Let's do this or add a ticket

Reply:
We can't invalidate the queries yet because they don't use React Query at the moment (we're still relying on `useVaiUser` for the time being).
I'll create a ticket and reference it there 👍